examples/modular-transformers/modeling_dummy.py

```diff
@@ -201,8 +196,8 @@ def forward(self, x, position_ids):
 
 def rotate_half(x):
```

Comment:
@LysandreJik a function that exists is properly updated! This was not the case before!